Services/Music_Portal.Services.Interfaces/Models/ArtistsLastFm.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System.Collections.Generic;
+
+namespace Services.Interfaces.Models
```

Comment:
The namespace should start with the Project name and then internal folders and e.c. like
`namespace Music_Portal.Services.Interfaces.Models`